pubspec.yaml

```diff
@@ -21,3 +22,10 @@ dev_dependencies:
   ffigen: ^2.4.2
   lints: ^1.0.1
   test: ^1.17.5
+
+flutter:
+  plugin:
```

Comment:
https://flutter.dev/docs/development/packages-and-plugins/developing-packages#plugin-platforms

We should specify the desktop platforms here as well.

Reply:
Added!

Reply:
I see you added more changes with this too, did you miss them before? 🙈🙈

Reply:
Actually, flutter on desktop works without these platform keys 😅 as I'm not adding any platform specific code myself. But, pub.dev may get angry 😶.

But, as I've added them now, I've to commit few boilerplate code also. Otherwise it will not work. 😬 That's why.

Reply:
Ah, gotchya 👌